.env.example

```
@@ -51,6 +56,7 @@ COINCUBE_ENABLE_PASSKEY=0
 # but still needs RP_ID below to match the ceremony domain.
 # Default (unset): http://localhost:8080/passkey
 COINCUBE_PASSKEY_CEREMONY_URL=
+COINCUBE_ENABLE_PASSKEY=1
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The COINCUBE_ENABLE_PASSKEY key is duplicated

(DuplicatedKey)